server/lobbyconnection.py

```diff
@@ -841,11 +841,14 @@ async def command_game_join(self, message):
             })
             return
 
+        if game.init_mode != InitMode.NORMAL_LOBBY:
```

Comment:
I disagree. If there are ever any more added (which I think is pretty unlikely anyways) I doubt that we will want people to be able to join this way. And if we do, we can change adjust the code when that happens. Better to fail fast than to have more bugs arise because of people joining lobbies they shouldn't be able to.

Reply:
I see. Well, i'm for bullet proof code. Leave it as it is.